lib/src/kbasesearchengine/main/IndexerWorker.java

```diff
@@ -531,7 +531,7 @@ private void indexObject(
                 logger.logInfo("[Indexer]   " + toVerRep(rule.getGlobalObjectType()) +
                         ", parsing time: " + parsingTime + " ms.");
                 long t3 = System.currentTimeMillis();
-                indexObjectInStorage(guid, timestamp, isPublic, obj, rule,
+                indexObjectInStorage(guid, timestamp, obj, rule,
```

Comment:
Should be an IndexerWorkerTest for public/private

Reply:
Have added indexerworker tests for public/private